xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/Applications.xml

```diff
@@ -692,66 +696,24 @@ require(['scriptaculous/effects'], function() {
     float: left;
     width: 33%;
   }
-
-  /* no responsive design for colibri */
-  .skin-colibri .panel-width-Small .applicationsPanel li {
-    float: none;
-    width: inherit;
-  }
 }
 
 @media (min-width: 768px) and (max-width: 992px) {
   .panel-width-Small .applicationsPanel &gt; li {
     float: left;
     width: 20%;
   }
-
-  /* no responsive design for colibri */
-  .skin-colibri .panel-width-Small .applicationsPanel li {
-    float: none;
-    width: inherit;
-  }
-}
-
-/* for colibri */
-
-.skin-colibri .panel-width-Small .applicationsPanel {
-  margin: 0;
-}
-
-.skin-colibri .panel-width-Small .applicationsPanel li {
-  padding: 15px 0;
-  -moz-border-radius: 7px;
-  -webkit-border-radius: 7px;
-  border-radius: 7px;
-  margin: 1px 0;
-}
```

Comment:
Not sure why you removed those changes?

Reply:
It was just cleaning up old code related to colibri, which we don't support anymore since 6.4 .
https://extensions.xwiki.org/xwiki/bin/view/Extension/Colibri%20Skin
 It was moved to contrib and IMO this code has little use being in the code base. Since this PR needed an update of the SSX, I thought it would be okay to clean it up a bit here directly.
Did I get something wrong?